src/gui/sdlmain.cpp

```diff
@@ -1924,23 +1936,61 @@ static bool DetectResizableWindow()
 #endif // C_OPENGL
 }
 
+static bool detect_shader_fixed_size()
+{
+#if C_OPENGL
+	if (sdl.desktop.want_type != SCREEN_OPENGL)
+		return false;
+
+	const std::string sname = get_glshader_value();
+	return (sname == "crt-fakelottes-flat" || sname == "crt-easymode-flat");
+#else
+	return false;
+#endif // C_OPENGL
+}
```

Comment:
This has a bit of a code-smell to it, given we're having to hard-code properties about items defined somewhere else in the code.

It would be nice for these properties to be stapled to the shaders, in the underordered map.
Maybe as a `pair` with these properties defined as an `enum`; or even make the shader a struct.
**shader:**
- name (const std::string)
- code (const char *)
- fixed_size (const bool)

This would also give us the ability take this property in dynamically for user-defined shader files (but in a subsequent PR).  Because right now, a user providing a fixed-size shader via filename/path would need the same name hardcoded here..).

Reply:
Hmm, this should probably be a new API function, e.g. `RENDER_GetSelectedShaderFlags`…

Let's do this after 0.76.0. - I added a sticky note with this task to 0.77.0 release.